### PR TITLE
added delete button itineraries.js

### PR DESCRIPTION
**Pull request by Jeffchiucp**
#32 added delete button for itineraries

### CODE REVIEW

File: views/index.jade [hunk 1]
```diff
@@ -3,16 +3,22 @@ extends layout
...
+        div.col-sm-6.col-sm-offset-3.itinerary-form
             // DISPLAY FORM ONLY WHEN USER IS LOGGED IN
             //- if (currentUser)
             .list-group
                 each itinerary, i in itineraries
-                    a.list-group-item(href='/itineraries/#{itinerary.id}/')
-                        h4.list-group-item-heading #{itinerary.name}
-                        p.list-group-item-text #{itinerary.date}
-                        p.list-group-item-text #{itinerary.description}
+                   .itinerary-list-container
```
Comment (anselb) [inline views/index.jade]: What is the point of these divs and classes on 14 and 15? They don't seem to be needed.

File: views/index.jade [hunk 2]
```diff
@@ -3,16 +3,22 @@ extends layout
...
+                      .itinerary-list-container-body
+                        a.list-group-item(href='/itineraries/#{itinerary.id}/')
+                            h4.list-group-item-heading #{itinerary.name}
+                            p.list-group-item-text #{itinerary.date}
+                            p.list-group-item-text #{itinerary.description}
+                        form.itinerary-delete
```
Comment (anselb) [inline views/index.jade]: After loading the page, the delete buttons do not look like they are in the right place. Please double check
screen shot 2018-01-15 at 10 48 39 pm

**Comment by Jeffchiucp** [comment]
Open the issue on #34

**Comment by Jeffchiucp** [comment]
Just updated the branch, fixed issue on #34 #35

**Comment by Jeffchiucp** [comment]
please review the new code. Thanks.

**Comment by Jeffchiucp** [comment]
screen shot 2018-01-16 at 3 30 32 pm

File: public/javascripts/routes-script.js [hunk 3]
```diff
@@ -35,4 +35,18 @@ $(document).ready(function(){
         thisForm.closest('.media').remove();
       })
     });
+
+    $('.itinerary-delete').submit(function(e){
```
Comment (anselb) [inline public/javascripts/routes-script.js]: This AJAX call appears not to work when the delete button is clicked.

Comment (Jeffchiucp) [reply public/javascripts/routes-script.js]: issue was resolved!

**Pull request merged**